assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/NoPricesBanner.tsx

```diff
@@ -4,11 +4,21 @@ import { __ } from '@wordpress/i18n';
 import { Banner } from '@application/ui/display';
 
 import AddDefaultPricesButton from '../buttons/AddDefaultPricesButton';
+import { TicketPriceCalculatorProps } from './TicketPriceCalculator';
 
-const NoPricesBanner = () => {
-	const description = __(
-		'Click the button below to load your default prices into the calculator. Additional ticket price modifiers can be added or removed. Click the save button below to assign which dates this ticket will be available for purchase on.'
+interface Props extends Pick<TicketPriceCalculatorProps, 'standalone'> {}
+
+const NoPricesBanner: React.FC<Props> = ({ standalone }) => {
+	let description = __(
+		'Click the button below to load your default prices into the calculator. Additional ticket price modifiers can be added or removed.'
 	);
+
+	if (standalone) {
+		description =
+			description +
+			__(' Click the save button below to assign which dates this ticket will be available for purchase on.');
```

Comment:
```suggestion
		description += __(' Click the save button below to assign which dates this ticket will be available for purchase on.');
```

Reply:
Also, it's better not to add that space at the beginning of the translation string. Translators can easily miss that.